src/main/java/createQuiz/CreateQuizInteractor.java

```diff
@@ -44,11 +44,11 @@ public void createQuiz(CreateQuizInputData inputData) {
         // Fail views
         if (Quiz.getTracker().containsKey(quizName)) {
             CreateQuizOutputData outputData = new CreateQuizOutputData(null,
-                    "This quiz name exists already. Failed to create new quiz.");
+                    "Quiz name already exists. Please choose a different name.");
```

Comment:
better error message

Reply:
Better as in a good job or I could do better?

Reply:
as in good job